src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
btw: just tried "modern compatibility", and my hexchat on mac didn't connect:

* Connection failed ((336032784) error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure)